faq.md

```diff
@@ -156,18 +156,86 @@ No. Monetization by the original manufacturer is what determines whether a produ
+- If you're a maintainer, and you _do_ monetise your FOSS codebase, you may still be exempted, depending on exactly how you are monetizing the codebase and your participation in it. For more detail, see [tmp-133c](#faq-tmp-133c).
+
+Second, even if the CRA does ultimately apply to you, penalties for solo and small-team maintainers are unlikely to be severe. For more detail, see [tmp-133d](#faq-tmp-133d).
+
+As a result, we would strongly urge you _not_ to shut down any open source projects (or your participation in those projects) just because of the CRA.
+
+> Status: ⚠️ [Draft][]
+| GitHub issue(s): [#133](https://github.com/orcwg/cra-hub/issues/133)
+</details>
+
 <details>
     <a name="q-am-i-subject-to-the-cra-if-i-only-contribute-to-an-open-source-project"></a>
     <summary><strong><a name="faq-tmp-17" href="#faq-tmp-17">tmp-17.</a> Am I subject to the CRA if I only contribute to an open source project?</strong></summary>
 
-No. Contributions to an open source project are explicitely not in scope of the CRA. See [Recital 18][]: 
+No. Contributions to an open source codebase are explicitely not in scope of the CRA. See [Recital 18][]: 
 
 > This Regulation does not apply to natural or legal persons who contribute with source code to products with digital elements qualifying as free and open-source software that are not under their responsibility.
 
 > Status: ⚠️ [Draft][]
 | GitHub issue(s): [#17](https://github.com/orcwg/cra-hub/issues/17)
 </details>
 
+<details>
+    <a name="q-am-I-subject-if-I-dont-monetise"></a>
+    <summary><strong><a name="faq-tmp-133b" href="#faq-tmp-133b">tmp-133b.</a> Am I subject to the CRA if I maintain, but do not monetise, an open source project?</strong></summary>
+
+If you are the maintainer of an open source codebase, and you do not monetise it, then the CRA _does not_ require you to do _anything_.
+
+The CRA applies
+> only in relation to products... supplied ... in the course of a _commercial activity_
+([Recital 15][], emphasis added)
+
+And it states that
+
+> the provision of ... free and open-source software that are not monetised by their manufacturers _should not be considered to be a commercial activity_
+([Recital 18][], emphasis added)
+
+
+> Status: ⚠️ [Draft][]
+| GitHub issue(s): [#133](https://github.com/orcwg/cra-hub/issues/133)
+</details>
+
+<details>
+    <a name="q-am-I-subject-if-I-do-monetise"></a>
+    <summary><strong><a name="faq-tmp-133c" href="#faq-tmp-133c">tmp-133c.</a> Am I subject to the CRA if I maintain and monetise an open source project?</strong></summary>
+
+If you are the maintainer of an open source codebase, and you do monetise it, then the CRA may apply to you, since you may be participating in a "commercial activity". 
+
+However, there are at least two significant exceptions that may allow you to take money for your work without being subject to the CRA.
```

Comment:
Perhaps the negative example of “technical support services” could/should also be listed here?

Reply:
Moved to #225